pkg/manifest/manifest_test.go

```diff
@@ -23,7 +23,10 @@ func TestPodManifestCanBeRead(t *testing.T) {
 	Assert(t).AreEqual("hello", string(manifest.ID()), "Id read from manifest didn't have expected value")
 	Assert(t).AreEqual(manifest.GetLaunchableStanzas()["app"].Location, "hoisted-hello_def456.tar.gz", "Location read from manifest didn't have expected value")
 	Assert(t).AreEqual("hoist", manifest.GetLaunchableStanzas()["app"].LaunchableType, "LaunchableType read from manifest didn't have expected value")
-	Assert(t).AreEqual("app", manifest.GetLaunchableStanzas()["app"].LaunchableId, "LaunchableId read from manifest didn't have expected value")
+
+	if manifest.GetLaunchableStanzas()["app"].LaunchableId != "app" {
```

Comment:
what if  `manifest.GetLaunchableStanzas()["app"].LaunchableId.String()` were to be used in the assert instead?